src/OrchardCore/Orchard.Localization.Abstractions/ILocalizationFileLocationProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Localization.Abstractions
+{
+    public interface ILocalizationFileLocationProvider
```

Comment:
You mentioned `IViewLocationExpander`. It provides alternative paths for a view matching values in `ViewLocationExpanderContext`. But we need something different, we need to get all paths of .po files. There isn't any relation to `ViewLocationExpanderContext`, so I can't see any way, how we can reuse it.